docs/articles/documentation/recipes/running-tests-locally.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+title: Running Tests Locally
+permalink: /documentation/recipes/running-tests-locally.html
+---
+# Running Tests Locally
+
+There are many ways to run your tests locally using TestCafe, but in this recipe we will focus on a simple one that uses a few other packages and can be easily integrated into your workflow. To do this, go through the following steps.
+
+* [Step 1 - Install TestCafe and create tests](#step-1---install-testcafe-and-create-tests)
+* [Step 2 - Install http-server and concurrently](#step-1---install-testcafe-and-create-tests)
+* [Step 3 - Add the `test` script to package.json](#step-5---add-the-test-script-to-packagejson)
```

Comment:
Please correct the link for the second step too. It should be
`#step-2---install-http-server-and-concurrently`